features/conversation-list/conversation-list.screen-header.tsx

```
@@ -89,33 +95,33 @@ function HeaderTitle() {
-    <HStack style={$titleContainer}>
+    <HStack style={themed($titleContainer)}>
       <ProfileAvatar />
       <DropdownMenu
         style={themed($dropDownMenu)}
         onPress={onDropdownPress}
         actions={[
-          ...currentProfiles?.map((profile) => ({
-            id: profile.id,
-            title: profile.name,
-            image: currentAccountInboxId === profile.xmtpId ? "checkmark" : "",
+          ...preferredDisplayInfos?.filter(Boolean).map((profile) => ({
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)